### PR TITLE
Add "Priority feature requests" to Enterprise plan description

### LINKED ISSUE #11643: Add priority on feature requests to pricing page
## Hello!

- Vote on this issue by adding a 👍 reaction
- If you want to implement this feature, comment to let us know (we'll work with you on design, scheduling, etc.)

## Issue details

We need to add to our Enterprise and Business Critical offerings on the pricing page (just add to Enterprise since its included in Business Critical) that we also offer priority on feature requests. This is a top thing our Enterprise+ customers come to us for and want to highlight it,. 

### Affected area/feature

Pricing page

Under Enterprise 
(checkmark) Priority feature requests

### PR DESCRIPTION
**Pull request by thoward**
Fixes #11643 (see for context)

### CODE REVIEW

**Comment by thoward** [comment]
@meagancojocar this is an older issue, is this still something we want to include on the pricing page?

File: content/pricing/_index.md [hunk 1]
```diff
@@ -66,6 +66,7 @@ tiers:
                   - Secrets versioning
                   - Audit logs
                   - 12x5 Enterprise Support available
+                  - Priority feature requests
```
Comment (meagancojocar) [inline content/pricing/_index.md:69]:
```suggestion
                  - Priority feature implementation
```

**Review by meagancojocar — APPROVED** [review]
Thank you for doing this! Love it!

File: content/pricing/_index.md (hunk 1 above)
Comment (meagancojocar) [reply content/pricing/_index.md:69]: @thoward I did not realize auto merge was on!

**Pull request merged**